pkg/local-disk-manager/handler/localdisk/localdisk.go

```diff
@@ -205,6 +204,10 @@ func (ldHandler *Handler) FilterDisk(ldc *v1alpha1.LocalDiskClaim) bool {
 		NoPartition().
 		IsNameFormatMatch().
 		GetTotalResult()
+	if !matched {
+		ldHandler.EventRecorder.Event(ldc, v1.EventTypeWarning, v1alpha1.LocalDiskClaimEventReasonAssignFail, ldHandler.filter.GetMessage())
+	}
```

Comment:
Is there too many events on the LDC? If one event is recorded for each LD.

I suggest that the disk allocation is completed and all disks are classified and recorded, including those due to inappropriate partitions and disk types. In this case, only one record is recorded at a reconcile

Reply:
Yes, you are right.I try to modify it.